ksp_plugin/interface_flight_plan.cpp

```diff
@@ -44,7 +44,7 @@ ksp_plugin::Burn FromInterfaceBurn(Plugin const* const plugin,
   return {burn.thrust_in_kilonewtons * Kilo(Newton),
```

Comment:
Plenty of Instant() left in this file. Perhaps make the constructor of Point private, befriending Frame and a J2000 function?